### PR TITLE
Added carnivore, rebel/mutineer, reverie's killer becomes them

### PR DESCRIPTION
**Pull request by TronAndRey**
Added Carnivore, Rebel/Mutineer, Reverie's killer becomes them. Haven't tested at all, may have missed some needed code.

### CODE REVIEW

File: Modules/CustomRolesHelper.cs [hunk 1]
```diff
@@ -245,6 +245,8 @@ public static RoleTypes GetDYRole(this CustomRoles role) // Role has a kill butt
             CustomRoles.PlagueBearer => RoleTypes.Impostor,
             CustomRoles.Pestilence => RoleTypes.Impostor,
             CustomRoles.Agitater => RoleTypes.Impostor,
+            CustomRoles.Mutineer2 => RoleTypes.Impostor,
```
Comment (Tommy-XL) [inline Modules/CustomRolesHelper.cs:248]: Why are they named as Mutineer and Mutineer2?

File: Modules/CustomRolesHelper.cs [hunk 2]
```diff
@@ -1540,8 +1557,7 @@ public static bool IsRevealingRole(this CustomRoles role, PlayerControl target)
             ((role is CustomRoles.Marshall) && target.AllTasksCompleted()) ||
             ((role is CustomRoles.Workaholic) && (Options.WorkaholicVisibleToEveryone.GetBool())) ||
             ((role is CustomRoles.Doctor) && (Options.DoctorVisibleToEveryone.GetBool())) ||
-            ((role is CustomRoles.Bait) && (Options.BaitNotification.GetBool()) && ParityCop.ParityCheckBaitCountType.GetBool()) ||
-            ((role is CustomRoles.President) && President.CheckPresidentReveal[target.PlayerId] == true));
+            ((role is CustomRoles.Bait) && (Options.BaitNotification.GetBool()) && ParityCop.ParityCheckBaitCountType.GetBool()));
```
Comment (Tommy-XL) [inline Modules/CustomRolesHelper.cs:1560]: You have removed the required code

File: Modules/OptionHolder.cs [hunk 3]
```diff
@@ -2219,18 +2232,17 @@ public static void Load()
...
-               .SetValueFormat(OptionFormat.Multiplier);
-           SpeedBoosterTimes = IntegerOptionItem.Create(18011, "SpeedBoosterTimes", new(1, 99, 1), 5, TabGroup.OtherRoles, false)
-           .SetParent(CustomRoleSpawnChances[CustomRoles.SpeedBooster])
-               .SetValueFormat(OptionFormat.Times); */
-        /*   SetupRoleOptions(18100, TabGroup.OtherRoles, CustomRoles.Glitch);
-           GlitchCanVote = BooleanOptionItem.Create(18110, "GlitchCanVote", true, TabGroup.OtherRoles, false)
-               .SetParent(CustomRoleSpawnChances[CustomRoles.Glitch]); */
-        //     Divinator.SetupCustomOption();
-        Admirer.SetupCustomOption();
+     /*   SetupRoleOptions(18000, TabGroup.OtherRoles, CustomRoles.SpeedBooster);
```
Comment (Tommy-XL) [inline Modules/OptionHolder.cs:2235]: Admirer.SetupCustomOption(); should be in experimental role

**Comment by Tommy-XL** [comment]
> Never added Rebel to CheckGameEndPatch, but that can easily be added.

If it needs to be added, it is better to add it now, because there may be bugs

**Comment by 0xDrMoe** [comment]
Please fix this and let me know when it's marked to be ready to pull

**Comment by Tommy-XL** [comment]
Closed because there are conflicts in the code that need to be fixed

**Pull request closed without merging**